site/docs/zh-CN/core/bundle.md

```
@@ -0,0 +1,97 @@
+└── bundle-manifest.json      # 参考元数据（externals、chunks 等）
+```
+
+完整细节见[产物结构参考](https://github.com/eggjs/egg/blob/next/tools/egg-bundler/docs/output-structure.md)。
+
+## 运行
+
+被识别为 **external** 的包不会被内联，必须与产物一起安装。最简单的方式是把应用的
+`package.json` 拷贝到 `worker.js` 旁边，并安装生产依赖：
+
+```bash
+$ cd dist-bundle
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)